### PR TITLE
Feature1

### PR DESCRIPTION
**Pull request by juramdi**
Review this code

### CODE REVIEW

**Comment by leopadilla** [comment]
Accept my work please

**Comment by juramdi** [comment]
Add the file3 please

**Comment by leopadilla** [comment]
Ok

**Comment by leopadilla** [comment]
Done

File: file3 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+line 1
+line 2
+line 3
```
Comment (juanleoz) [inline file3]: Add the line 4 please

File: file2 [hunk 2]
```diff
@@ -0,0 +1,2 @@
+line 1
```
Comment (juanleoz) [inline file2]: Add line 0 please

File: file1 [hunk 3]
```diff
@@ -0,0 +1,2 @@
+line 1
+line 2
```
Comment (juanleoz) [inline file1]: Too here

File: file3 [hunk 4]
```diff
@@ -0,0 +1,3 @@
+line 1
+line 2
```
Comment (juanleoz) [inline file3]: And here

**Pull request merged**